tle/cogs/handles.py

```diff
@@ -349,6 +350,28 @@ async def set(self, ctx, member: discord.Member, handle: str):
         users = await cf.user.info(handles=[handle])
         await self._set(ctx, member, users[0])
 
+    @handle.command(brief="Refreshing the handles of users")
+    @commands.has_any_role("Admin", "Moderator")
+    async def refresh(self, ctx):
+        await ctx.send("Please wait...")
+        registeredUsers = cf_common.user_db.get_cf_users_for_guild(ctx.guild.id)
+        for user in registeredUsers:
+            discordID = user[0]
+            currHandle = user[1].handle
+            oldURL = 'https://codeforces.com/profile/' + currHandle
+            newHandle = str((requests.get(oldURL).url)).split('/')[-1]
```

Comment:
Won't this requests.get run into some kind of CF API rate limiting? Do we want to put a `sleep(0.2)` at the end of the loop?

Reply:
Yes, I have updated that part. Added a sleep(0,2) for the part where we make the API call.